docs/configuration.md

```diff
@@ -3241,6 +3241,29 @@ private final case class A2(a: Int)
     extends ADT
 ```
 
+If you prefer to use the order based on the one
+[partially](https://docs.scala-lang.org/style/declarations.html#modifiers)
+specified in the Scala Style Guide, you can use the corresponding preset:
+
+```scala mdoc:scalafmt
+rewrite.rules = [SortModifiers]
+rewrite.sortModifiers.preset = styleGuide
```

Comment:
Would probably be awesome to have this as a global style that would setup up most needed things. Just random thought

Reply:
hmm... i don't know what that entails, to be honest. the style guide hasn't been updated or maintained in a while, but i am happy to let someone who feels strongly about this, to take a stab.